sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -11,7 +11,7 @@ The Azure Key Vault certificates client library enables programmatically managin
 Install the Azure Key Vault certificates client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet install Azure.Security.KeyVault.Certificates -IncludePrerelease
+dotnet add package Azure.Security.KeyVault.Certificates -IncludePrerelease
```

Comment:
I don't think the `IncludePrerelease` is a valid `dotnet add package` flag

Reply:
Let me check, I blindly converted here.

Reply:
It should be `--prerelease`. I will correct. Apologies.

Reply:
I will also test each one.

Reply:
`prerelease` is .NET 5 specific flag. Let's use the `--version` parameter for now.

```suggestion
dotnet add package Azure.Security.KeyVault.Certificates --version 4.2.0-beta.3
```